WooCommerceShared.podspec

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# This podspec is code generated base on the podspec.erb template.
+# To make changes, please edit the template.
+
+Pod::Spec.new do |s|
+  s.name          = 'WooCommerceShared'
+  s.version       = '0.0.1'
+
+  s.summary       = 'Shared components used for the iOS and Android WooCommerce Apps.'
+  s.description   = "#{s.summary} It's a React Native library project."
+
+  s.homepage      = 'https://github.com/woocommerce/WooCommerce-Shared'
+  # s.license       = { type: 'MPL', text: File.read('LICENSE') }
+  s.author        = { 'The WooCommerce Mobile Team' => 'mobile@wordpress.org' }
+
+  s.platform      = :ios, '13.0'
+  s.swift_version = '5.0'
+
+  s.source_files = 'WooCommerceShared.xcframework'
+
+  s.source = {
+    http: 'https://cdn.a8c-ci.services/woocommerce-shared/376403148c39d452177120034e90795830027498/WooCommerceShared.xcframework.zip'
+  }
+
+  s.vendored_frameworks = 'WooCommerceShared.xcframework'
+end
```

Comment:
I went for code generating and uploading the podspec because that was the working approach I found for Gutenberg.

I was happy to see that the `zip` archive worked out of the box, without the shenanigans of adding an extra folder and `dummy.txt` I have in Gutenberg for the `tar.gz`.

That makes me hopeful that maybe using the standard `pod git: commit:` syntax might work. If that's the case, we ought to revert my code generation and use the runtime podspec evaluation that was implemented before (see #5)